CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 <!-- next version -->
 
+## 0.112.0
+
+### 💡 Enhancements 💡
+
+- `auto-instrumentation`: Support configuring Java auto-instrumentation when runtime configuration is provided from configmap or secret. (#1814)
+  This change allows users to configure JAVA_TOOL_OPTIONS in config map or secret.
+  The operator in this case set another JAVA_TOOL_OPTIONS that references the original value 
+  e.g. `JAVA_TOOL_OPTIONS=$(JAVA_TOOL_OPTIONS) -javaagent:/otel-auto-instrumentation-java/javaagent.jar`.
```

Comment:
@pavolloffay I think this is incorrect. Your change works when users set `JAVA_TOOL_OPTIONS` directly, it doesn't work if it's set from a ConfigMap or Secret via `envFrom`.

Reply:
@yuriolisa do you prefer to change it inline here or shall I make a PR to main to edit the original change log? 

> This change allows users to configure JAVA_TOOL_OPTIONS in config map or secret.

change to 

> This change allows users to configure JAVA_TOOL_OPTIONS in configmap or secret when the name of the variable is defined in the pod spec.

Reply:
I will change it.